### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by kenan-rhoton**
**Description**

Add a template for future Pull Requests.

**Issues Resolved**

N/A

**Details**

Guidelines to make it easier to review pull requests, as well as understand the code being submitted.

**Notes**

Special attention should be given to the Pull Request titles, in particular I'm not very sold on "Notes".

### CODE REVIEW

**Review by Masclins — DISMISSED** [review]
It feels a bit weird that this PR doesn't follow the template.

I assume this will be shown whenever one opens a PR.

Thanks.

**Comment by kenan-rhoton** [comment]
It does follow it, I just accidentally clicked submit before filling in the text :upside_down_face:

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+**Issues Resolved**
+
+List of issues the Pull Request fixes
+
+**Details**
+
+More in-depth explanation about the code submitted and its purpose.
+
+**Notes**
```
Comment (Masclins) [inline PULL_REQUEST_TEMPLATE.md]: What about "Hotspots"?

Comment (kenan-rhoton) [reply PULL_REQUEST_TEMPLATE.md]: I was considering "Highlights" maybe?

**Review by Masclins — APPROVED** [review]

**Pull request merged**